src/main/java/com/redhat/red/offliner/cli/Options.java

```diff
@@ -42,6 +42,12 @@
 
     private static final Integer DEFAULT_THREADS = 4 * Runtime.getRuntime().availableProcessors();
 
+    @Option( name = "-H", aliases = { "--header" }, metaVar = "HEADER",
+             usage = "File to supply option values normally specified on command line which will cover other options declared in cli. " +
+                     "This supports ini-style format, uses option alias as the param name per line, " +
+                     "example:\nno-metadata (no need to specify its boolean value)\ndownload=./repository\nFILES=manifest.txt" )
+    private File headerFile;
```

Comment:
Will this header file options override the options in cli? If it will, I think we should add some warnings in this option description.

Reply:
@ligangty, sorry I missed this comment, yes, it will cover the options, and the first line in the description indicates that: 'File to supply option values normally specified on command line which will cover other options declared in cli.'

Reply:
From my reading of this, it appears that specifying `--header` will cause offliner to completely ignore all other CLI options, which would probably surprise most users. I think most users would also expect a command-line option to override the configured options in the file header.

Also, I intended this header to literally be a header section in the top of the manifest file, instead of a separate file. That way, the user only needs one file (plus offliner itself) to launch a repository download. I think we need this to be something that we pull out of the top of the file, maybe down to a line that contains `---` (3 or more single dash characters).